src/style.css

```
@@ -1 +1,12 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.form-field:last-child {
+  margin-bottom: 0;
+}
+
+.fieldset__margin {
```

What happens if I will wanna use paddings? Do I need to rename and replace this class (in all places🥲) with `fieldset__padding`?

```suggestion
.fieldset__margin {
```